src/members/Members.jsx

```diff
@@ -1,21 +1,79 @@
-import React from "react";
+import React, { useState, useContext, useEffect } from "react";
+import { Table } from "react-bootstrap";
+import { AuthContext } from "../AuthContext";
+import { BASE_API } from "../config";
 
 export default function Members() {
-    return (
-        <div className="container-fluid" id="members">
-          <div className="top">
-            <h1>
-              This is the Public page for Members list
-            </h1>
-          </div>
-          <div className="middle">
-            <h2>The site is currently under construction...</h2>
-          </div>
-          <div className="bottom">
-            <h3>
-                Thank you for your patience and understanding.
-            </h3>
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [members, setMembers] = useState([]);
+  const { access_token } = useContext(AuthContext);
+
+  const requestMembersList = {
+    method: "GET",
+    headers: {
+      "Authorization": `Bearer ${access_token}`,
+      "Accept": "application/json",
+      "Content-Type": "application/json",
+    },
+  };
+
+  useEffect(() => {
+    fetch(`${BASE_API}/users`, requestMembersList)
+      .then(async response => {
+        const data = await response.json();
+        if (response.ok) {
+          return setMembers(data);
+        }
+        setErrorMessage(data.message);
+      })
+      .catch(error =>
+        setErrorMessage("The service is temporarily unavailable, please try again later.")
```

Comment:
Why does this not use the SERVICE_UNAVAILABLE_ERROR?

Reply:
Yup. Fixed it.